rectools/models/sasrec.py

```diff
@@ -52,6 +55,100 @@ def forward(self, seqs: torch.Tensor, timeline_mask: torch.Tensor, attn_mask: to
         raise NotImplementedError()
 
 
+class SessionEncoderDataPreparatorBase:
```

Comment:
Please move this class under `# ####  --------------  Data Processor  --------------  #### `

Reply:
fixed